Use more compact font

rosano opened the pull request:
This is aesthetic but also a bit more compact as the letters are less
wide.

Simon-Laux commented:
no visible changes on firefox on linux

rosano commented:
Before:

<img width="760" alt="Screen Shot 2022-12-15 at 19 15 11" src="https://user-images.githubusercontent.com/1680612/207936764-0dd18436-adfe-4a12-828c-b29456e0b1bc.png">

After:
<img width="760" alt="Screen Shot 2022-12-15 at 19 15 09" src="https://user-images.githubusercontent.com/1680612/207936787-f2e79c85-593f-45b2-8258-52f2b9101b62.png">

The first point about messaging anyone fits on one line.

rosano commented:
> no visible changes on firefox on linux

The fonts might not be available? But nothing was removed, only an extra option added if available.

r10s commented:
there is still some letter spacing i added that time for h1...h3 - is this still on purpose?

apart from that, "Lucida Grande" seems not to be considered a web-safe font (random link: https://www.cssfontstack.com) "Lucida Grande" is mostly mac.

rosano commented:
> there is still some letter spacing i added that time for h1...h3 - is this still on purpose?

Wasn't aware of it but could be left in or taken out. I think it's too compressed without the spacing, but with Verdana it's too wide.

> "Lucida Grande" seems not to be considered a web-safe font (random link: https://www.cssfontstack.com) "Lucida Grande" is mostly mac - not even windows.

Updated to use Arial everywhere and Tahoma for headings.

missytake commented:
On my xubuntu with Firefox this change seems to make it worse right now.

Status quo:

![Screenshot_2022-12-16_10-10-35](https://user-images.githubusercontent.com/57442838/208063800-16294c77-ac26-4ec2-9019-743020874701.png)

With this PR:

![Screenshot_2022-12-16_10-10-42](https://user-images.githubusercontent.com/57442838/208063813-ae3caac2-2bf2-4b9f-928b-75da7b523680.png)

rosano commented:
> is this what Tahoma is supposed to look like?

Weird because I only see `sans-serif` fonts in the whole project. Can you let me know what's the computed style/font for you? I can only imagine this happening because of browser defaults or custom user CSS. Here's what I see:

<img width="648" alt="Screen Shot 2022-12-16 at 12 36 23" src="https://user-images.githubusercontent.com/1680612/208090036-4d26e547-d50c-4f39-b6a6-1aa590b6d849.png">

missytake commented:
I think it's because I don't have Tahoma and there is no fallback font configured?

![Screenshot_2022-12-16_12-58-36](https://user-images.githubusercontent.com/57442838/208093592-67a60601-9c11-42c4-88e1-c9f39665f5f4.png)

rosano commented:
> I think it's because I don't have Tahoma and there is no fallback font configured?

I thought it would use the body font… Would you mind editing either in the browser or in source to see if `font-family: Tahoma, inherit;` works? Just because the issue is on your machine. Worst case, we can use `font-family: Tahoma, sans-serif;`

missytake commented:
`font-family: Tahoma, inherit;` doesn't work, but `font-family: Tahoma, sans-serif;` works :)

missytake reviewed (approved):
works now :)

r10s commented:
the body text is harder to read with this pr.

this is just because the font is overall smaller now - even if the pt size is not changed. is this on purpose?

to get the effective same font-size, it needs to be increased from 15px to 16px and the line-height can be decreased from 1.7em to 1.6em - side by side we would have the same font-size but a bit more compact.

overall, i am +/-0 with this pr,  maybe it is good to have sth. fresh after these years.

rosano commented:
> needs to be increased from 15px to 16px and the line-height can be decreased from 1.7em to 1.6em - side by side we would have the same font-size but a bit more compact.

Done. I increased the heading size to give more contrast to the body text.

> overall, i am +/-0 with this pr, maybe it is good to have sth. fresh after these years.

I have (perhaps incorrectly) perceived various desires to improve the site design and I'm trying to do small incremental changes rather than big ones, so it might be a more noticeable improvement when they're combined. Perhaps I should also just propose non-incremental changes so that the difference is more obvious.

r10s commented:
> and I'm trying to do small incremental changes rather than big ones

this is very welcome!

and i think, after some forth and back, font is nice now. different, but nice :)

> so it might be a more noticeable improvement when they're combined. 

good point.

r10s commented on _sass/_base.scss:
```diff
@@ -155,17 +155,21 @@ p.blogdate {
-    line-height: 1.7em;
+    font-family: Arial, Verdana, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.6em;
```
```suggestion
    line-height: 1.5em;
```

did i say 1.6em? triying it over, i mean 1.5em :) with that, and the increased font-size, height and spacing are same as before (and aesthetic :), with, of course, the more compact font.
